apps/chat_web/lib/chat_web/live/chat_live.html.heex

```diff
@@ -5,7 +5,10 @@
         id={message.id}
         class={"mb-4 py-4 px-4 border rounded-xl #{if message.sender == :bot, do: "bg-gray-200"}"}
       >
-        <p class="text-gray-800 line-clamp-2 break-words"><%= message.text %></p>
+        <p class="text-gray-800 line-clamp-2 break-words">
+          <%= raw Earmark.as_html!(message.text) %>
```

Comment:
There are some alternative ways to do this I suppose, and most Elixir style guides seem to discourage omitting parentheses in function calls (as to not introduce confusion between function calls and local variables). But since we do not have a style guide for this project, I will leave this decision up to you whether any of my suggestions are actually better.

```suggestion
          <%= raw(Earmark.as_html!(message.text))%>
```
or
```suggestion
          <%= message.text |> Earmark.as_html!() |> raw()%>
```

Reply:
Thanks, I used the second version, I didn't think it could be used like that here.